deviceModeInit.js

```diff
@@ -101,7 +127,12 @@ var rudderTracking = (function () {
       .catch((error) => {
         console.log("Error occurred while updating cart:", error);
       });
-    identifyUser();
+    isClientSideIdentifierEventsDisabled().then(response => {
```

Comment:
```suggestion
    isClientSideIdentifierEventsDisabled().then(response => {
      if (!response) {
          identifyUser();
      }
    })
```

Reply:
There are two more places from where we are making identify call. Hence I made a global value `disableClientIdentifierEvents`, assigned value to that and providing the check through same.